samples/super_resolution/README.md

```diff
@@ -0,0 +1,29 @@
+# Super Resolution
+
+The pipeline shows the use of super-resolution neural networks to improve the quality and resolution of the input video. 
+
+We prepared ONNX versions (using standard `torch.onnx.export`) of two lightweight NinaSR models ninasr_b0 and ninasr_b1 with different scales (x2/3/4) from the [TorchSR repository](https://github.com/Coloquinte/torchSR). The demo uses input video in 360p format and model ninasr_b0 with scale 3 to output video in 1080p format. The model and scale can be changed in the module configuration [file](module.yml).
+
+The demo is prepared for use on dGPU. Separately, the module can be run on the Jetson Xavier platform also. 
```

Comment:
Don't understand use of `separately` here.

Reply:
Separate = without other performance-impacting demo services (e.g. AO-RTSP). But I'd better delete that confusing sentence.